scripts/run_classify.py

```diff
@@ -0,0 +1,71 @@
+"""
+Script to run mutation detection pipeline
+
+"""
+import pathlib
+import subprocess
+
+import sys; sys.path.append('.')
+import config as cfg
+from data_models import DataModel
+
+pathway_map = {
+    cfg.pathway_data.joinpath('canonical_mapped.tsv').resolve(): 'canonical_pathways',
+    cfg.pathway_data.joinpath('oncogenic_mapped.tsv').resolve(): 'oncogenic_pathways',
+    cfg.pathway_data.joinpath('randomized_pathways.tsv').resolve(): 'random_pathways'
+}
+
+k_vals = [10, 20, 50, 100, 200]
+algorithms = DataModel.list_algorithms()
+# gene list from BioBombe paper, just do these for now
+genes = ['TP53', 'PTEN', 'PIK3CA', 'KRAS', 'TTN']
+
+def run_compression(algorithm, k):
+    if alg == 'plier':
+        for pathway_file, pathway_dir in pathway_map.items():
+            cmd = ['python', '1.compress_given_z.py',
+                   '-a', alg, '-k', str(k), '-v',
+                   '-p', str(pathway_file),
+                   '-o', str(cfg.models_dir.joinpath(pathway_dir).resolve())]
+            print('Running: {}'.format(' '.join(cmd)))
+            subprocess.check_call(cmd)
+            cmd = ['python', '1.compress_given_z.py',
+                   '-a', alg, '-k', str(k), '-s', '-v',
+                   '-p', str(pathway_file),
+                   '-o', str(cfg.models_dir.joinpath(pathway_dir).resolve())]
+            print('Running: {}'.format(' '.join(cmd)))
+            subprocess.check_call(cmd)
+    else:
+        cmd = ['python', '1.compress_given_z.py',
+               '-a', alg, '-k', str(k), '-v',
+               '-o', str(cfg.models_dir.joinpath('canonical_pathways').resolve())]
+        print('Running: {}'.format(' '.join(cmd)))
+        subprocess.check_call(cmd)
+        cmd = ['python', '1.compress_given_z.py',
+               '-a', alg, '-k', str(k), '-s', '-v',
+               '-o', str(cfg.models_dir.joinpath('canonical_pathways').resolve())]
```

Comment:
Depending on whether you intend to write more scripts depending on it, you may want to create an importable function in `compress_given_z.py` instead of calling it as a subprocess each time (or maybe not if you think you'll end up calling it multiple times in paralell)

Reply:
I'll think about this for the future. I was trying to follow the BioBombe convention of having numbered, procedural scripts to run the experiments (the `1.` in the filename prevents it from being imported) and moving reusable/repeated code to other importable modules, so I'll keep it like this for now.

If I do end up writing more scripts that depend on this, I'll create importable functions.